app/controllers/collections_controller.rb

```diff
@@ -49,6 +49,15 @@ def update
     end
   end
 
+  def destroy
+    collection = Collection.find(params[:id])
+    authorize! collection
```

Comment:
@jcoyne does this validate via the policy (to ensure the collection is a draft)?  and same question for the work

Reply:
Yes, it's using the `CollectionPolicy#destroy?` method you wrote which checks `first_draft?`